pages/common/git-log.md

```diff
@@ -14,6 +14,10 @@
 
 `git log --oneline`
 
+- Show all commits and branches in a graph format:
+
+`git log --oneline --abbrev-commit --decorate --all --graph`
```

Comment:
I think you can do without the `--abbrev-commit` option. Its good not to overload an example with too many options.

Also, please explain the `--decorate` and `--all` options. `--oneline` is already done above.